scripts/bb-test-prepare.sh

```diff
@@ -22,6 +22,13 @@ SUDO="sudo -E"
 
 set -x
 
+# Attempt to blacklist the buildslave to prevent it
+# from being targeted by the oom-killer
+if test -d "$BB_DIR"; then
```

Comment:
Testing for the existence of the file here rather than the directory would probably make more sense.  It would also be good to check for the existence of the proc file since I'm not sure which kernel version added this .  This can all be best effort.